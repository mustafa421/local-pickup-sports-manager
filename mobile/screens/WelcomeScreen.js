import React from "react";
import { StyleSheet, Text, View } from "react-native";
import { Button } from "react-native-elements";

const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: "#fff",
    alignItems: "center",
    justifyContent: "space-evenly"
  }
});

export default function WelcomeScreen({ navigation }) {
  return (
    <View style={styles.container}>
      <Text>Welcome to Local Pickup Sports Manager!</Text>

      <Button
        title="Sign in with Facebook"
        backgroundColor="#3B5998"
        onPress={() => navigation.navigate("auth", { loginMethod: "fb" })}
      />
      <Button
        title="Sign in with Google"
        backgroundColor="#f2a60c"
        onPress={() => navigation.navigate("auth", { loginMethod: "google" })}
      />

    </View>
  );
}
