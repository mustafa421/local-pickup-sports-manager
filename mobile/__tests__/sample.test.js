import "react-native";

import React from "react";
import renderer from "react-test-renderer";
import WelcomeScreen from "../screens/WelcomeScreen";

describe("sample test suite", () => {
  test("adds 1 + 2 to equal 3", () => {
    expect(1 + 2).toBe(3);
  });

  test("test", () => {
    expect(1 + 2).toBe(3);
  });

  test("snapshot", () => {
    const tree = renderer.create(<WelcomeScreen />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
