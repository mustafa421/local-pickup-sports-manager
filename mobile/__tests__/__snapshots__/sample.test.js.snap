// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sample test suite snapshot 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "flex": 1,
      "justifyContent": "space-evenly",
    }
  }
>
  <Text>
    Welcome to Local Pickup Sports Manager!
  </Text>
  <View
    style={
      Array [
        Object {
          "marginLeft": 15,
          "marginRight": 15,
        },
        undefined,
        undefined,
        undefined,
      ]
    }
  >
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={null}
    >
      <View
        pointerEvents="box-only"
        style={
          Array [
            Object {
              "alignItems": "center",
              "backgroundColor": "#9E9E9E",
              "flexDirection": "row",
              "justifyContent": "center",
              "padding": 19,
            },
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            Object {
              "backgroundColor": "#3B5998",
            },
            undefined,
            Object {
              "padding": 12,
            },
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
          ]
        }
      >
        <Text
          style={
            Array [
              Object {},
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              Array [
                Object {
                  "color": "white",
                  "fontSize": 20,
                },
                undefined,
                Object {
                  "fontSize": 17.5,
                },
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
              ],
            ]
          }
        >
          Sign in with Facebook
        </Text>
      </View>
    </View>
  </View>
  <View
    style={
      Array [
        Object {
          "marginLeft": 15,
          "marginRight": 15,
        },
        undefined,
        undefined,
        undefined,
      ]
    }
  >
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={null}
    >
      <View
        pointerEvents="box-only"
        style={
          Array [
            Object {
              "alignItems": "center",
              "backgroundColor": "#9E9E9E",
              "flexDirection": "row",
              "justifyContent": "center",
              "padding": 19,
            },
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            Object {
              "backgroundColor": "#f2a60c",
            },
            undefined,
            Object {
              "padding": 12,
            },
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
          ]
        }
      >
        <Text
          style={
            Array [
              Object {},
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              Array [
                Object {
                  "color": "white",
                  "fontSize": 20,
                },
                undefined,
                Object {
                  "fontSize": 17.5,
                },
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
              ],
            ]
          }
        >
          Sign in with Google
        </Text>
      </View>
    </View>
  </View>
</View>
`;
